### PR TITLE
Added info about chef cookbook

### PR DESCRIPTION
**Pull request by AlexJakeGreen**
Hi Mykola,

I created a cookbook to get the script installed with chef provisioning, and added appropriate info into the README file.

Please approve this pull request :)

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -46,7 +46,9 @@ to use it. Run
   make
 
 to make installation files. Then copy gather script somewhere you want
-to have it, preferably in some directory from PATH. gather reads its
+to have it, preferably in some directory from PATH. Or you can look into
+a chef cookbook https://github.com/AlexJakeGreen/gatherit-cookbook for
+automatical installation of the script. gather reads its
```
Comment (trociny) [inline README]: Could you please add a section "2.1 Installing with Chef" (or something like this) in the end of "2.Installation and configuration" section and add this info there instead?

**Comment by AlexJakeGreen** [comment]
Hi again.
The section "2.1 Installing with Chef" has been added.
Since the cookbook is already published on Chef Supermarket - an open-source community platform -  I just added a link to there so anyone can easily find instructions how to install it.
Thanks

**Comment by trociny** [comment]
looks good. Can you please squash these two commits into one?

```
git rebase -i HEAD~2
git push -f
```

(google for "git interactive rebase" if it is something new for you and you need details).

**Comment by AlexJakeGreen** [comment]
Ooops! Squashed.
Thanks. I appreciate your help.

**Pull request merged**